### ISSUE TITLE
Cause an error when calling `clpy.random.rand()` with empty argument

### ISSUE THREAD

**Issue by neko-suki** [fixstars/clpy#169]
When I call `clpy.random.rand()` with empty argument, it cause an error. I tested it on the Secondary machine. 

- Following is an error log. 
```
>>> import clpy
>>> clpy.random.rand()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/username/.pyenv/versions/clpy/lib/python3.6/site-packages/clpy-2.1.0b1-py3.6-linux-x86_64.egg/clpy/random/sample.py", line 33, in rand
    return random_sample(size=size, dtype=dtype)
  File "/home/username/.pyenv/versions/clpy/lib/python3.6/site-packages/clpy-2.1.0b1-py3.6-linux-x86_64.egg/clpy/random/sample.py", line 154, in random_sample
    return rs.random_sample(size=size, dtype=dtype)
  File "/home/username/.pyenv/versions/clpy/lib/python3.6/site-packages/clpy-2.1.0b1-py3.6-linux-x86_64.egg/clpy/random/generator.py", line 182, in random_sample
    RandomState._init_kernel(tmp_seed_array, self.seed_array)
  File "clpy/core/elementwise.pxi", line 628, in clpy.core.core.ElementwiseKernel.__call__
  File "clpy/util.pyx", line 39, in clpy.util.memoize.decorator.ret
  File "clpy/core/elementwise.pxi", line 479, in clpy.core.core._get_elementwise_kernel
  File "clpy/core/elementwise.pxi", line 15, in clpy.core.core._get_simple_elementwise_kernel
  File "clpy/core/elementwise.pxi", line 44, in clpy.core.core._get_simple_elementwise_kernel
  File "clpy/core/carray.pxi", line 158, in clpy.core.core.compile_with_cache
  File "clpy/core/carray.pxi", line 181, in clpy.core.core.compile_with_cache
clpy.backend.ultima.exceptions.UltimaRuntimeError: Return code:1
STDERR:
/tmp/111220.696859143.cpp:17:65: error: no matching function for call to 'get_CArrayIndex_1'
__attribute__((annotate("clpy_elementwise_tag")))T x;x = seed + get_CArrayIndex_1(&x_info, &_ind);;
                                                                ^~~~~~~~~~~~~~~~~
/home/username/.pyenv/versions/clpy/lib/python3.6/site-packages/clpy-2.1.0b1-py3.6-linux-x86_64.egg/clpy/../clpy/core/include/clpy/carray.clh:386:15: note: candidate function not viable: no known conversion from 'CArray_0 *' to 'const CArray_1 *const __restrict' for 1st argument
static size_t get_CArrayIndex_1(const CArray_1* const __restrict__ info, const CIndexer_1* const __restrict__ _ind) {
              ^
1 error generated.
Error while processing /tmp/111220.696859143.cpp.
```

- The observation with `cupy` is as follows. 

```
>>> import cupy
cu>>> cupy.random.rand()
array(0.97502676)
>>> cupy.random.rand().shape
()
>>> cupy.random.rand().size
1
```

**Comment by neko-suki** [comment]
This may be solved when `random_tests` https://github.com/fixstars/clpy/issues/109 is passed.

**Comment by ghost** [comment]
solved in #259
